### DISCUSSION TITLE
DisableDefaultUI / Hiding map controls?

### DISCUSSION THREAD

**Discussion by gbrits** [capacitor-community/google-maps#198] [General]
Is it possible currently to hide the controls on a map or is this something that is yet to be implemented? 
I noticed the config has no allowance for the `DisableDefaultUI` attribute.

**Comment by tafelnl** [comment]
Yes, this is possible. You can specify several preferences for controls and appearance in the `createMap` and `updateMap` methods. See the docs for this: https://capacitor-community.github.io/google-maps/#/api?id=mapcontrols
